cli/README.md

```diff
@@ -0,0 +1,55 @@
+# Consul Kubernetes CLI
+This repository contains a CLI tool for installing and operating [Consul](https://www.consul.io/) on Kubernetes. 
+**Warning** this tool is currently experimental. Do not use it on Consul clusters you care about.
+
+## Installation & Setup
+Currently the tool is not available on any releases page. Instead clone the repository and run `go build -o bin/consul-k8s`
+and proceed to run the binary.
+
+## Commands
+* [consul-k8s install](# consul-k8s install)
+
+### consul-k8s install
+```
+Usage: consul-k8s install [flags]
+
+ Install Consul onto a Kubernetes cluster.
+
+Flags:
+
+
+  -auto-approve
+ 	Skip confirmation prompt.
+
+  -dry-run
+ 	Validate installation and return summary of installation.
+
+  -config-file,-f=<string>
+ 	Path to a file to customize the installation, such as Consul Helm chart values file. Can be specified multiple times.
+
+  -name=<string>
+ 	Name of the installation. This will be prefixed to resources installed on the cluster.
+
+  -namespace=<string>
+ 	Namespace for the Consul installation. Defaults to “consul”.
+
+  -preset=<string>
+ 	Use an installation preset, one of default, demo, secure. Defaults to "default".
+
+  -set=<string>
+ 	Set a value to customize. Can be specified multiple times. Supports Consul Helm chart values.
+
+  -set-file=<string>
+      Set a value to customize via a file. The contents of the file will be set as the value. Can be specified multiple times. Supports Consul Helm chart values.
+
+  -set-string=<string>
+      Set a string value to customize. Can be specified multiple times. Supports Consul Helm chart values.
+
```

Comment:
@kschoche's comment: Can we add a bit more information in the readme to define the order of precedence for any of these flags.
Example:
`consul-k8s-cli install -f val.yaml -preset=secure -set="global.tls.enabled=false"` 
I can guess that this will probably break the install somehow due to helm charts themselves, but it is not clear to me what the order of precedence for these flags in general. What if I set them twice, to different values, or set conflicting flags.
I think we can solve most of these questions (for this whole file) by just expanding on how things work in the Readme file.

Reply:
Added to the readme a description of the precedence ordering!